### PR TITLE
fix: Agent/Worker comm using files in Python and Cpp

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**

### CODE REVIEW

**Comment by dbrasseur-aneo** [comment]
Cla bot not giving the comment...

File: packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h [hunk 1]
```diff
@@ -89,7 +84,8 @@ class TaskHandler {
    * @param data The result data
    * @return A future containing a vector of ResultReply
    */
-  std::future<armonik::api::grpc::v1::agent::ResultReply> send_result(std::string key, absl::string_view data);
+  std::future<armonik::api::grpc::v1::agent::NotifyResultDataResponse> send_result(std::string key,
```
Comment (aneojgurhem) [inline packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h:87]: It is unfortunate that it changes the API. I think you should wrap this message somehow

File: packages/cpp/tools/restart-build-env.sh
Comment (aneojgurhem) [inline packages/cpp/tools/restart-build-env.sh:1]: why does this change in this PR ?

File: packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h (hunk 1 above)
Comment (aneojgurhem) [reply packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h:87]: I'd return nothing and throw if there is an issue

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**